Client/Frontend/Login Management/LoginDataSource.swift

```diff
@@ -69,10 +65,9 @@ class LoginDataSource: NSObject, UITableViewDataSource {
             cell.textLabel?.text = login.hostname
             cell.detailTextColor = UIColor.theme.tableView.rowDetailText
             cell.detailTextLabel?.text = login.username
-            cell.accessoryType = .disclosureIndicator
-            if let breaches = viewModel.userBreaches {
-                if breaches.contains(login) {
-                    cell.textLabel?.textColor = UIColor.systemRed
+            if let breaches = viewModel.userBreaches, breaches.contains(login) {
```

Comment:
Should we change the `userBreaches` and `breachIndexPath` collections to a `Set` type since we are calling contains on them in a UI function that we want to run quickly?  Do they need to be indexed arrays for any reason?